### DISCUSSION TITLE
Public repo is inaccessible in the app

### DISCUSSION THREAD

**Discussion by bwhaley** [community/community#42508] [Mobile]
When I visit the 1debit/alternat repo in the app, it just says "Try again" and will not open. Loads fine in a browser.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.93.0 (85890385)
Device: iPhone 12 Pro (iOS 16.1.1)
</details>

**Comment by jfullstackdev** [comment]
as a workaround while this is not fixed yet, just log in in your mobile browser and you will have same experience as GitHub Web
